src/wp-includes/class-wp-block-pattern-categories-registry.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * Blocks API: WP_Block_Pattern_Categories_Registry class
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @since 5.5.0
+ */
+
+/**
+ * Class used for interacting with block pattern categories.
+ */
+final class WP_Block_Pattern_Categories_Registry {
+	/**
+	 * Registered block pattern categories array.
+	 *
+	 * @var array
+	 */
+	private $registered_categories = array();
+
+	/**
+	 * Container for the main instance of the class.
+	 *
+	 * @var WP_Block_Pattern_Categories_Registry|null
+	 */
+	private static $instance = null;
+
+	/**
+	 * Registers a pattern category.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @param string $category_name       Pattern category name.
+	 * @param array  $category_properties Array containing the properties of the category: label.
+	 * @return boolean True if the pattern was registered with success and false otherwise.
+	 */
+	public function register( $category_name, $category_properties ) {
+		if ( ! isset( $category_name ) || ! is_string( $category_name ) ) {
+			$message = __( 'Block pattern category name must be a string.' );
+			_doing_it_wrong( __METHOD__, $message, '5.5.0' );
```

Comment:
```suggestion
			_doing_it_wrong( __METHOD__, __( 'Block pattern category name must be a string.' ), '5.5.0' );
```

The extra variable seems to be unnecessary